### ISSUE TITLE
Seems to be broken

### ISSUE THREAD

**Issue by xstasy** [xCausxn/nodebb-plugin-twitch#1]
Using the new redactor and it wont embed twitch currently.

**Comment by xCausxn** [comment]
Will look into this issue asap :)

EDIT: Just tested on latest nodebb - http://prntscr.com/7z0wv4

**Comment by xstasy** [comment]
Yeah, I disabled [nodebb-plugin-composer-redactor](https://github.com/NodeBB/nodebb-plugin-composer-redactor#readme) and it worked, I wish you'd support this though.
I think they're planning to ship it as the default editor in a later version.

**Comment by xCausxn** [comment]
Well the way my plugin works is the same way as the youtube one, however if its shipped as default then i can look into it a bit more :)
